README.md

```diff
@@ -20,7 +20,7 @@ Repository | NIO 2 (Swift 5.2+)
 [https://github.com/apple/swift-nio-transport-services][repo-nio-transport-services] <br> first-class support for macOS, iOS, tvOS, and watchOS | `from: "1.0.0"`
 [https://github.com/apple/swift-nio-ssh][repo-nio-ssh] <br> SSH support | `.upToNextMinor(from: "0.2.0")`
 
-NIO 2.29.0 and older support Swift 5.0+.
+NIO 2.29.0 and older support Swift 5.0+, NIO 2.39.0 and older support Swift 5.2+.
```

Comment:
All happy with that. Mind adding some official wording regarding security updates. Let's assume a security vulnerability in NIO 2.29.0 and newer gets identified. Who will get patches? I assume that at this point it'll be three updates: One for 2.29.x, one for 2.39.x and one for latest. Correct?

Reply:
Assuming the vulnerability exists in all three, I think that's the policy, yes.

Reply:
Sounds good. I think this should just be stated explicitly. It's clear (from elsewhere in that doc) that NIO 1.x won't get security updates anymore (as of May 2022) but that doesn't make it explicit that NIO 2.x _will_ get them for the time being.